test/basetest/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+configure_file(__init__.py __init__.py COPYONLY)
+configure_file(compat.py compat.py COPYONLY)
+configure_file(exceptions.py exceptions.py COPYONLY)
+configure_file(hooks.py hooks.py COPYONLY)
+configure_file(meta.py meta.py COPYONLY)
+configure_file(task.py task.py COPYONLY)
+configure_file(testing.py testing.py COPYONLY)
+configure_file(utils.py utils.py)
```

Comment:
I guess it's critical that COPYONLY doesn't appear here, as that's how the `${..}` get a value in this file?

Reply:
Exactly, that is the file where the variables are updated, which is not possible when `COPYONLY` is set.